src/files/XmlFile.ts

```diff
@@ -176,6 +184,16 @@ export class XmlFile {
                             Position.create(lineIndex, idx),
                             Position.create(lineIndex, idx + 10)
                         );
+                        //calculate the rane of the component's name (if it exists)
+                        let match = /(.*?name\s*=\s*(?:'|"))(.*?)('|")/.exec(this.lines[lineIndex]);
```

Comment:
Const

Reply:
yeah, might as well. This is a pretty big codebase, so haven't flipped the eslint switch yet to enforce this. I think I'll do this once we get your plugin PR merged, and the ternary PR merged.